config/dev.json

```diff
@@ -26,5 +26,6 @@
   "terraDockerVersionsFile": "terra-docker-versions-dev.json",
   "terraDockerImageBucket": "terra-docker-image-documentation-dev",
   "b2cBillingPolicy": "b2c_1a_signup_signin_billing_dev",
-  "terraDeploymentEnv": "dev"
+  "terraDeploymentEnv": "dev",
+  "cwdsUrlRoot": "https://cwds.dsde-dev.broadinstitute.org"
```

Comment:
I've only added the dev URL so far; is there any reason not to include urls for staging and prod?  If I don't, I'll have to deal with the lack of this value in the code, which would be removed when the config for those environments is updated.

Reply:
I think its fine to add these, they are already live